src/AppContainer.tsx

```diff
@@ -123,6 +126,26 @@ const AppContainer: FunctionComponent<Props> = ({ onSetWorkspaceInfo, onReconnec
         onSetWorkspaceInfo(workspaceInfo)
     })
 
+    const workspaceSubfeedForReducer = useMemo((): AppendOnlyLog => {
```

Comment:
We need to wrap the workspaceSubfeed in workspaceSubfeedForReducer because the messages in the subfeed are of the form `{action: x}`, whereas the reducer just takes the actions (ie `x`)